src/main/java/org/elasticsearch/index/query/CommonTermsQueryBuilder.java

```diff
@@ -163,6 +197,19 @@ public CommonTermsQueryBuilder lowFreqMinimumShouldMatch(String lowFreqMinimumSh
         return this;
     }
 
+    public String lowFreqMinimumShouldMatch() {
+        return this.lowFreqMinimumShouldMatch;
+    }
+
+    public CommonTermsQueryBuilder disableCoord(Boolean disableCoord) {
```

Comment:
given that we had no setter for this, I would take the chance to change the type to the primitive `boolean` here, not sure booleans are a good fit for the complication of null->default